GitCommands/Settings/AppSettings.cs

```diff
@@ -424,6 +424,12 @@ public static bool ShowGitStatusForArtificialCommits
             set => SetBool("showgitstatusforartificialcommits", value);
         }
 
+        public static bool SortByAuthorDate
+        {
+            get => GetBool("sortbyauthordate", true);
```

Comment:
Default should ba false, much slower appearance for larger repos

Reply:
Fixed.

Reply:
@gerhardol I believe I've addressed all the comments - is there anything else I need to do?